bin/maze-runner

```diff
@@ -31,23 +31,25 @@ class MazeRunnerEntry
   # Removes Maze Runner specific args from the array, as these will cause Cucumber to error.
   def remove_maze_runner_args
     remove = [
```

Comment:
Not something to do now, but I think this could be replaced with `remove = Maze::Option.constants`

Reply:
Updated on the next PR.